rialto_airflow/harvest/pubmed.py

```diff
@@ -93,7 +92,52 @@ def harvest(snapshot: Snapshot, limit=None) -> Path:
     return jsonl_file
 
 
-def pmids_from_orcid(orcid):
```

Comment:
this method didn't go away, i just added a type check for the return value, but the diff moved this change down below the new `fill_in` method